README.md

```diff
@@ -100,6 +100,11 @@ router.use(
 
 Next, add a route the user of your app will be redirected to once the authorization process has finished. This example uses the freshly obtained credentials to send a message to the user's Slack workspace:
 
+Ensure you install the official Slack SDK:
+```bash
+npm install --save @slack/web-api
+```
+
```

Comment:
Move this block above the "add a route" block.

Reply:
Fixed